modules/logicadBidAdapter.js

```diff
@@ -0,0 +1,68 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {BANNER} from '../src/mediaTypes';
+
+const BIDDER_CODE = 'logicad';
+const ENDPOINT_URL = 'https://pb.ladsp.com/adrequest/prebid';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid: function (bid) {
+    return !!(bid.params && bid.params.tid);
+  },
+  buildRequests: function (bidRequests, bidderRequest) {
+    const requests = [];
+    for (let i = 0, len = bidRequests.length; i < len; i++) {
+      const request = {
+        method: 'POST',
+        url: ENDPOINT_URL,
+        data: JSON.stringify(newBidRequest(bidRequests[i], bidderRequest)),
+        options: {},
+        bidderRequest
+      };
+      requests.push(request);
+    }
+    return requests;
+  },
+  interpretResponse: function (serverResponse, bidderRequest) {
+    serverResponse = serverResponse.body;
+    const bids = [];
+    if (!serverResponse || serverResponse.error) {
+      return bids;
+    }
+    serverResponse.seatbid.forEach(function (seatbid) {
+      bids.push(seatbid.bid);
+    })
+    return bids;
+  },
+  getUserSyncs: function (syncOptions, serverResponses) {
+    if (serverResponses.length > 0 && serverResponses[0].body.userSync &&
+      syncOptions.pixelEnabled && serverResponses[0].body.userSync.type == 'image') {
+      return [{
+        type: 'image',
+        url: serverResponses[0].body.userSync.url
```

Comment:
To confirm - if you're returning multiple bids, this code would only grab the first bid's userSync pixel.  The other bids' userSync pixels would be left out.

Is that expected?

Reply:
Hi @jsnellbaker 

Thank you for the review.

Yes, it is as expected.

Only using the first bids will fulfill our requirements.